book/src/rpc/server.rs

```diff
@@ -15,7 +15,7 @@ impl BookService for BookServer {
         match queries::get_book_by_id(book_id as i32) {
             Ok(book) => Ok(book),
             Err(e) => {
-                log::error!("Error: {}", e);
+                log::warn!("Error (get_book): {}", e);
```

Comment:
I think we should talk about what events we consider to belong to a certain log level. For me a wrong client request would go as a `debug` message. It's something which can happen by accident and is not relevant for the normal run of the system. Such a default can pollute the logs quite a lot if someone has bad intentions.